packages/app-mobile/components/screens/Note.tsx

```diff
@@ -1206,6 +1391,61 @@ class NoteScreenComponent extends BaseScreenComponent {
 
 		const noteTagDialog = !this.state.noteTagDialogShown ? null : <NoteTagsDialog onCloseRequested={this.noteTagDialog_closeRequested} />;
 
+		// Different styles for when the note actions are active or not
+		const notesBarToggleIconStyle = this.props.showNotesBar ? this.styles().noteActionButtonIconActive : this.styles().noteActionButtonIcon;
+		const notesBarToggleStyle = this.props.showNotesBar ? this.styles().noteActionButtonActive : this.styles().noteActionButton;
+
+		const handleNoteActionsDrag = (gestureState: any) => {
+			const minY = 160;
+			const maxY = 0.8 * Dimensions.get('window').height;
+
+			let newY = gestureState.moveY;
+
+			if (newY < minY) {
+				newY = minY;
+			} else if (newY > maxY) {
+				newY = maxY;
+			}
+
+			this.noteActionsPositionY.setValue(newY - 162);
+		};
+
+		// Pan responder that handles making the note actions draggable.
+		// The note actions need to be draggable, because they could
+		// potentially obstruct some portion of a note's content
+		this.noteActionsDragResponder = PanResponder.create({
+			onMoveShouldSetPanResponder: () => true,
```

Comment:
```suggestion
			// Only start dragging after moving at least 10px — this prevents clicks from dragging instead
			// of triggering onPress events
			onMoveShouldSetPanResponder: (_evt, gestureState) => {
				return Math.abs(gestureState.dx) > 10 || Math.abs(gestureState.dy) > 10;
			},
```

The StackOverflow post I linked above recommends `return gestureState.dx != 0 && gestureState.dy != 0`. This doesn't work on all devices. For example, with my tablet's stylus, it is very difficult to click the button instead of dragging if using `return gestureState.dx != 0 && gestureState.dy != 0`.

Reply:
Thanks for the fix. I don't use a device with a stylus, so I couldn't have spotted this.

Reply:
> @Tolu-Mals, if you could address personalizedrefrigerator's comments I think we can merge.

Alright, working on it.